package.json

```diff
@@ -41,18 +42,29 @@
   },
   "devDependencies": {
     "@semantic-release/git": "^9.0.0",
+    "cypress": "^5.1.0",
     "ejs-lint": "^1.1.0",
     "prettier": "^2.0.5",
-    "semantic-release": "^17.1.1"
+    "ps-tree": "^1.2.0",
+    "semantic-release": "^17.1.1",
+    "start-server-and-test": "^1.11.3",
+    "wait-on": "^5.2.0"
   },
   "release": {
     "branches": [
-      "main",
-      "next"
+      {
+        "name": "main",
+        "channel": "canary",
+        "prerelease": "canary"
+      },
+      {
+        "name": "release"
+      }
```

Comment:
This should be more in line with what we wanted to do in the release config. We want master to be a canary prerelease, and will cut main releases from the release branch.